mm2src/mm2_db/src/indexed_db/indexed_cursor.rs

```diff
@@ -210,17 +209,17 @@ pub trait WithFilter: Sized {
 }
 
 #[async_trait]
-pub trait CollectCursor<Table: DeserializeOwned> {
+pub trait CollectCursor<Table: TableSignature> {
     async fn collect(self) -> CursorResult<Vec<(ItemId, Table)>>;
 }
 
 #[async_trait]
-impl<Table: DeserializeOwned + 'static, T: CollectCursorImpl<Table> + Send> CollectCursor<Table> for T {
+impl<Table: TableSignature, T: CollectCursorImpl<Table> + Send> CollectCursor<Table> for T {
     async fn collect(self) -> CursorResult<Vec<(ItemId, Table)>> { self.collect_impl().await }
 }
 
 #[async_trait]
-pub trait CollectCursorImpl<Table: DeserializeOwned + 'static>: Sized {
+pub trait CollectCursorImpl<Table: TableSignature>: Sized {
```

Comment:
Only used in inner module, no need to define `pub` for the trait(I know it was already like that before, can be nice to update it).

Reply:
Done